harness/determined/common/experimental/workspace.py

```diff
@@ -36,7 +37,12 @@ def __init__(
             self.id = _get_workspace_id_from_name(session, workspace_name)
         else:
             self.id = workspace_id
-        self.name = workspace_name
+
+        if workspace_name is None:
+            assert workspace_id is not None
+            self.name = _get_workspace_name_from_id(session, workspace_id)
```

Comment:
Just realized it is intended not to populate name from id, because of a failed [test](https://github.com/determined-ai/determined/blob/main/harness/tests/determined/common/experimental/test_workspace.py#L87). May I know why we want to do this?

Reply:
circling back to this, can we omit it from the PR? I don't see this being used here. Maybe we keep the `_get_workspace_name_from_id` function?

Reply:
I prefer if `_get_workspace_name_from_id` is not merged. `client.get_workspace` is a nice feature, but can absolutely be done separately.